minishift/dev_setup.md

```diff
@@ -110,6 +110,23 @@ You can try logging by hitting this in browser `http://f8ui-fabric8-services.${m
 Note: However if you are trying this first time you should approve your username from keycloak, so that you will be authenticated user.
 Also make sure to whitelist the domain which you are using for auth to work it as per expectation.
 
+#### Notification Service
+##### Deploying Notification Service
+
+To deploy Notification service, we have following make target which will deploy service.
+```bash
+make deploy-notification
+```
+
+Look for running pods using `oc get pods`. You should be able to see two pods(f8notification-*). First time it will take some time as it has download required container images.
+
+##### Mandrill API key
+If you have mandrill api key. Make sure you updated secrets data by key `madrill.api` from `f8notification.config.yaml` with base64 encoded value of your key.
```

Comment:
is there a risk that the `f8notification.config.yaml` file was committed and pushed with someone's Mandrill API key? If so, could we avoid that and maybe rely on env variable instead of a config file committed in the repo? or, since `minishift/openshift/f8notification.config.yaml` is an OpenShift template, we could use to generate the `secrets` manifest by including this environment variable

Reply:
yes, there is a risk. Let me findgenerate it using env.